app/factories/bulkrax/object_factory.rb

```diff
@@ -104,7 +104,7 @@ def run!
 
     def update
       raise "Object doesn't exist" unless object
-      destroy_existing_files if @replace_files && ![Collection, FileSet].include?(klass)
+      destroy_existing_files if @replace_files && ![Collection, CollectionResource, FileSet].include?(klass)
```

Comment:
What happens when CollectionResource is not defined?  We need to favor a class method to answer the following:

`Bulkrax.config.collection_classes` and `Bulkrax.config.file_set_classes`, as we can't hard-code constants that may not exist in other non-Hyku 6 implementations.

Reply:
ohhh yea, great point.

Reply:
this should be resolved in main with your changes now.